ProjectSettings/QualitySettings.asset

```diff
@@ -201,7 +201,7 @@ QualitySettings:
     skinWeights: 4
     textureQuality: 0
     anisotropicTextures: 2
-    antiAliasing: 2
+    antiAliasing: 0
```

Comment:
Probably shouldn't add this file (since it's one of the problem files)?
https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

Reply:
Didn't we agree to commit these Settings files?  I remember taking them off the .gitignore after you requested a change. (I'm down to add them to the .gitignore though so we can all use our own settings locally)

Reply:
Oh I was saying we don't commit these files unless we actually made changes to the settings! The changes here are just Unity bugs that are constantly changing it to whatever it's not, so even if we commit this, it'll just change back next time :(

But since if we actually change QualitySettings (which is definitely going to happen) and want everyone to get those changes, we probably shouldn't add it to the .gitignore.